### PR TITLE
Improve bundle size

### PR DESCRIPTION
**Pull request by MarcelGerber**
Bundle size detective Marcel is at it again! 🔍 

I found some (rather simple) ways to shave off 10% of our bundle size, or -240KB before brotli / -70KB after brotli.

### CODE REVIEW

File: explorer/ExplorerGrammar.ts [hunk 1]
```diff
@@ -27,12 +26,8 @@ export const ExplorerGrammar: Grammar = {
     table: {
         ...UrlCellDef,
         keyword: "table",
-        terminalOptions: OwidDatasets.map((keyword) => ({
-            keyword,
-            description: "",
-            cssClass: "",
-        })),
         valuePlaceholder: "",
+        regex: new RegExp(`(${UrlCellDef.regex?.source ?? ""}|^[\\w -()]+$)`), // URL or OWID dataset name
```
Comment (danielgavrilov) [inline explorer/ExplorerGrammar.ts:30]: Nice! That makes sense, we could probably drop this feature entirely at some point.

**Review by danielgavrilov — APPROVED** [review]
🙌 Woo! Let's free up some bandwidth on those network links!

Do you think we should also lower the [bundlewatch config](https://github.com/owid/owid-grapher/blob/c87a801b55458d6ba418d0c1395dc258bc4ce15c/.bundlewatch.config.json#L3-L6) to something like 1.8MB in order to catch accidental increases?

File: explorer/ExplorerGrammar.ts (hunk 1 above)
Comment (MarcelGerber) [reply explorer/ExplorerGrammar.ts:30]: By feature you mean the ability to refer to OWID datasets directly, right?
Makes sense to me. [The way it works might be quite brittle](https://github.com/owid/owid-grapher/blob/43151675072ecdeba0e2a5cbce637e9035252c37/explorer/ExplorerProgram.ts#L399-L402), anyway.

**Comment by MarcelGerber** [comment]
Nice, thank you for your review!

**Pull request merged**